backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -176,16 +180,17 @@ CREATE TABLE transactions(
         JSONB,
 
     -- Within a single block, two transactions cannot share the same index.
-    PRIMARY KEY (block_height, index)
+    PRIMARY KEY (block_height, block_index)
 );
 
+CREATE INDEX transactions_index_idx ON transactions (index);
```

Comment:
Not sure if we should just get rid of the block_height/block_index and only use the global index. :shrug:

Reply:
I removed it.